controller/scrape.go

```diff
@@ -30,6 +30,8 @@ import (
 
 // Holds list of experiments in a chaosengine
 var chaosExperimentList []string
+// Holds the chaosresult of the running experiment
+var chaosresultMap = make(map[string]bool)
```

Comment:
This is essentially an experimentStatusMap. Can we not call it chaosresultMap -- there can be confusion (w/ older approach of deriving metrics from chaosresult CR)